public/core.d.ts

```diff
@@ -2093,8 +2093,70 @@ declare namespace JMap {
      * JMap.Map.setMapRotationControlVisibility(false)
      * ```
      */
-    function setMapRotationControlVisibility(isVisible:boolean):void
+    function setMapRotationControlVisibility(isVisible: boolean): void
+
+    /**
+     * ***JMap.Map.isMapInfoControlVisible***
+     * 
+     * Returns true if the Map Info control is visible on the map.
+     * 
+     * @example ```ts
+     * 
+     * // returns true if control is displayed on the map
+     * JMap.Map.isMapInfoControlVisible()
+     * ```
+     */
+    function isMapInfoControlVisible(): boolean
   
+    /**
+     * ***JMap.Map.setMapInfoControlVisibility***
+     * 
+     * Change the Map Info control visibility on the map.
+     * 
+     * @param isVisible true to display the Map Info control, false to hide
+     * 
+     * @example ```ts
+     * 
+     * // will display the Map Info control on the map
```

Comment:
Je pense que JR veux qu'on utilise la 3ième personne du singulier, i.e. "Displays the Map Info....", même dans les commentaires inline des exemple, faudrait valider avec lui